get/linux.md

```diff
@@ -51,6 +51,17 @@ To install the libcurl development package you run:
 
     dnf install libcurl-devel
 
+## Immutable/Atomic Fedora distributions (i.e Silverblue, Kinoite, Sericea, Onyx, ...)
```

Comment:
```suggestion
## Immutable Fedora distributions
```

Then mention the alternatives in the section, not in the title. To keep the title short and easy to read.